### ISSUE TITLE
Update recipe to Python 3.6

### ISSUE THREAD

**Issue by majahn** [menpo/conda-mayavi#1]
Hi, thanks a lot for your great work in bringing Mayavi to Python 3!!! It's awesome... ;-)

This conda-recipe worked like charm for Python 3.5 on my Mac. Unfortunately, it does not work yet for Python 3.6:

> conda install -c menpo mayavi
Fetching package metadata .....................
Solving package specifications: ....

> UnsatisfiableError: The following specifications were found to be in conflict:
>  - mayavi
>  - python 3.6*

I'm a complete newbie to conda builds, so I'm not sure if adding ` - PYTHON_VERSION: 3.6` to the .travis.yaml file is enough or whether there is more to it... I'm happy to learn to contribute a PR next time ... ;-))

Thanks again.

Ciao, 
Marcus

**Comment by patricksnape** [comment]
Yes, I haven't gone through and made builds for Python 3.6 yet - but in theory it should be as simple as adding 3.6 to the build matrix.

**Comment by jan-janssen** [comment]
I tired it, but I get the following error: 
```
creating stub loader for tvtk/array_ext.cpython-36m-x86_64-linux-gnu.so
byte-compiling build/bdist.linux-x86_64/egg/tvtk/array_ext.py to array_ext.cpython-36.pyc
installing package data to build/bdist.linux-x86_64/egg
running install_data
----------------------------------------------------------------------
Building TVTK classes... vtkContextDevice2D: Ignoring method: Get/SetViewportRect
default: [1701860128, 1634890851, 925376620, 41], range: None
vtkContextDevice2D: Ignoring method: Get/SetViewportSize
default: [-1241160072, 11145], range: None
vtkFXAAOptions: Ignoring method: Get/SetDebugOptionValue
default: 0, range: None
vtkGeoTreeNode: Ignoring method: Get/SetStatus
default: 0, range: None
vtkIncrementalForceLayout: Ignoring method: Get/SetGravityPoint
default: [200.0, 200.0], range: None
Segmentation fault
```

**Comment by patricksnape** [comment]
This might have some useful information: http://www.math.univ-paris13.fr/~cuvelier/mainsu26.html

**Comment by patricksnape** [comment]
Or [this](https://github.com/enthought/mayavi/issues/457) actually

**Comment by patricksnape** [comment]
Looks like we need to build VTK 7.1.1

**Comment by mwaskom** [comment]
Do you know if VTK 7.1.1 is packaged anywhere for conda? Unfortunately the main anaconda folks seem uninterested in offering mayavi for python 3. It's awesome that this project is supporting it, but I'd really like to be using 3.6!

**Comment by constatza** [comment]
Any progress guys? Will we be able to work with Python 3.6 in the near future? or should I just downgrade it to 3.5?

**Comment by TinghuiWang** [comment]
Here is something you can try:

If you have Visual Studio installed, and Python Native Development Tools installed.

In Anaconda3, you can get VTK7.1.x from clinicalgraphics
conda install -c clinicalgraphics vtk 

Open up the VS20xx x64 Native Command Prompt, setting up PATH for python properly:
e.g. C:\Anaconda3\Scripts\activate.bat C:\Anaconda3

Then install mayavi using pip from source.
pip install mayavi

It compiles successfully with VS2017 and Python 3.6 in windows 10.

**Comment by constatza** [comment]
I will try it, thank you!
